database/migrations/sql/20181009_20_add_unique_start_time_constraint_volunteer_hours_log.sql

```diff
@@ -0,0 +1,8 @@
+/*
+ * Add unique start time constraint on volunteer_hours_log
+ *
+ * User should not be able to register multiple logs that have identical
+ * start times.
+ */
+ALTER TABLE volunteer_hours_log
```

Comment:
@eliascodes why is this being added as a migration?

Reply:
I think this was before we decided we could re-do the DB. I'll move it into the original migrations.